### PR TITLE
chore(Makefile): add gopath validation and targets to run nodes for a local dev network

### PR DESCRIPTION
**Pull request by kanishkatn**
## Changes

- Added a validation on `install` target to make sure GOPATH is set
- Added targets to run nodes for a local dev network

## Primary Reviewer

@timwu20

### CODE REVIEW

**Review by EclesioMeloJunior — APPROVED** [review]

**Review by kishansagathiya — COMMENTED** [review]
Not a big fan of using make targets as command short-hands

File: Makefile [hunk 1]
```diff
@@ -126,4 +126,35 @@ gossamer: clean build
 
 ## install: install the gossamer binary in $GOPATH/bin
 install: build
+ifndef GOPATH
+	$(error GOPATH is not set)
+endif
```
[the remark this answers is not in the document]
Comment (qdm12) [reply Makefile:131]: Why is this check needed?
I think documentation precises you need Go installed, and that should set `GOPATH` right?
As in, if we have this check, we would open the door to add a lot more checks such as, do you have `go` installed, the right `go version`, do you have `g++` installed for wasmer etc.

File: Makefile [hunk 2]
```diff
@@ -126,4 +126,35 @@ gossamer: clean build
...
+	./bin/gossamer --config ./chain/westend-local/config-charlie.toml
+
+start-alice: build
+	./bin/gossamer --config ./chain/westend-local/config-alice.toml --babe-lead
+
+start-bob: build
+	./bin/gossamer --config ./chain/westend-local/config-bob.toml
+
+start-charlie: build
+	./bin/gossamer --config ./chain/westend-local/config-charlie.toml
```
Comment (qdm12) [inline Makefile:160]: Noice 👍

File: README.md [hunk 3]
```diff
@@ -176,6 +176,51 @@ Then start the three hosts:
...
+make init-alice
+```
+
+```
+make init-bob
+```
+
+```
+make init-charlie
+```
```
Comment (qdm12) [inline README.md:193]: nit maybe merge these in a single code block?

File: README.md [hunk 4]
```diff
@@ -176,6 +176,51 @@ Then start the three hosts:
...
+make run-alice
+```
+
+```
+make run-bob
+```
+
+```
+make run-charlie
+```
```
Comment (qdm12) [inline README.md:208]: nit maybe merge these in a single code block?

File: README.md [hunk 5]
```diff
@@ -176,6 +176,51 @@ Then start the three hosts:
...
+make start-alice
+```
+
+```
+make start-bob
+```
+
+```
+make start-charlie
+```
```
Comment (qdm12) [inline README.md:222]: nit maybe merge these in a single code block?

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (qdm12) [reply Makefile:131]: See https://go.dev/doc/gopath_code

> The GOPATH environment variable specifies the location of your workspace. It defaults to a directory named go inside your home directory, so $HOME/go on Unix, $home/go on Plan 9, and %USERPROFILE%\go (usually C:\Users\YourName\go) on Windows. 

Let's remove it, I think you just had a fluke on your system and `GOPATH` didn't get set. In all my systems it's set automagically

**Pull request closed without merging**